host/xtest/pkcs11_1000.c

```diff
@@ -8392,6 +8414,12 @@ static CK_RV test_rsa_aes_wrap(ADBG_Case_t *c, CK_SESSION_HANDLE session,
 		  t->key.modulus_len },
 		{ CKA_PUBLIC_EXPONENT, (CK_VOID_PTR)t->key.pub_exp,
 		  t->key.pub_exp_len },
+		{ CKA_EXPONENT_1, (CK_VOID_PTR)t->key.exp1,
+		  t->key.prime2_len },
+		{ CKA_EXPONENT_2, (CK_VOID_PTR)t->key.exp2,
+		  t->key.prime2_len },
+		{ CKA_COEFFICIENT, (CK_VOID_PTR)t->key.coeff,
+		  t->key.coeff_len },
```

Comment:
These attributes relate the private keys whereas `wrapping_key_template` defines a public key.